torch/testing/_internal/common_methods_invocations.py

```diff
@@ -588,22 +588,38 @@ def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad, **kw
 
     return inputs
 
+def sample_inputs_addmm_non_fusible_nodes(op_info, device, dtype, requires_grad, **kwargs):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
+    alpha_beta = kwargs.get('alpha_beta', None)
+    tests_list = [
+        ((2, 3), (2, 2), (2, 3)),
+    ]
+    tests_require_resizing = [
+        ((1,), (2, 2), (2, 3)),
+        ((), (2, 2), (2, 3))
+    ]
+    test_cases = [*tests_list] if for_inplace_variant else [*tests_list, *tests_require_resizing]  # type: ignore
```

Comment:
Lists can just be added together or one list can `extend` the other

Reply:
Doing so avoids copying, but I understand that doesn't matter here 😄